examples/10_Perera_Science_Suzuki/example_Perera.ipynb

```diff
@@ -961,6 +959,8 @@
        "      <td>MeCN</td>\n",
        "      <td>4.764109</td>\n",
        "      <td>6.262059e+03</td>\n",
+       "      <td>NaN</td>\n",
+       "      <td>NaN</td>\n",
```

Comment:
What are these NaN values and the Unnamed columns that got added?

Reply:
I think this could be due to a pandas update? If I open the file in Numbers it shows two empty columns at the end. Updated to use `usecols` in the call to `read_excel`.